docs/source/integrations/paddlepaddle.rst

```diff
@@ -0,0 +1,21 @@
+###############
+ PaddlePaddle
+###############
+
+This topic describes how to integrate TensorBay dataset with PaddlePaddle Pipeline
+using the `MNIST Dataset <https://gas.graviti.cn/dataset/data-decorators/MNIST>`_ as an example.
+
+The typical method to integrate TensorBay dataset with PaddlePaddle is to build a "Segment" class
+derived from ``paddle.io.Dataset``.
+
+.. literalinclude:: ../../../docs/code/use_dataset_in_paddlepaddle.py
+   :language: python
+   :start-after: """Build a Segment class"""
+   :end-before: """"""
```

Comment:
This code block will contains the following code block, please add another end-before in the `use_dataset_in_paddlepaddle.py`.

Reply:
Done~